cocos/core/platform/view.ts

```diff
@@ -79,28 +79,18 @@ if (legacyCC.sys.os === legacyCC.sys.OS_IOS) { // All browsers are WebView
     __BrowserGetter.adaptationType = legacyCC.sys.BROWSER_TYPE_SAFARI;
 }
 
-if (WECHAT) {
-    __BrowserGetter.availWidth = () => {
-        return window.innerWidth;
-    };
-    __BrowserGetter.availHeight = () => {
-        return window.innerHeight;
-    };
```

Comment:
method`avalidWidth avaliHeight` not used anymore in `_initFrameSize`, cause method `_initFrameSize` is reimplemented in adapters repo